7a_temp_coop_munge/src/data_parsers/parse_mndow_coop_files.R

```diff
@@ -281,3 +299,53 @@ parse_Sand_Bay_All_2016 <- parse_Sand_Bay_all_2015 <- parse_Sand_Bay_all_2014 <-
   saveRDS(object = clean_sheet, file = outfile)
   sc_indicate(ind_file = outind, data_file = outfile)
 }
+
+parse_LeechMain2018 <-
+  parse_LeechWalkerBay2018 <-
+  parse_MilleLacs2018 <-
+  parse_Winnie2018 <-
+  parse_Rainy2018 <-
+  parse_Kab2018 <- function(inind, outind) {
```

Comment:
Whoa does this work?? So all these named function have the same structure? That's pretty cool

Reply:
Yes! I think I learned this from @wdwatkins. Originally there was one parser per file, so matching parser name to file name made sense. This gets around this issue!